common/templates/context_funcs.go

```diff
@@ -1632,6 +1632,67 @@ func (c *Context) tmplEditChannelTopic(channel interface{}, newTopic string) (st
 	return "", err
 }
 
+func (c *Context) tmplSetForumTags(channel interface{}, input interface{}) (string, error) {
```

Comment:
instead of repurposing this to remove forumtags, maybe create a separate template to removeForumTags, and removeAllForumTags. 

`set` should ideally just set tags. 

I would've personally preferred something like `addForumTags` and `removeForumTags` instead of set which modified the complete list of tags.

Reply:
`setRoles` has the same behaviour, it removes roles
the target has but weren't given.

Changing `setTags` to only add would be a bit of an
inconsistency.

Reply:
As Lzodd said, `setRoles` has the same behaviour and implementing another function to remove it wouldn't be necessary.
However, there are also add/removerole based methods. and I wouldn't be against implementing `addTag`/`removeTag` methods as well if you feel as though it would be useful to also have them